forge-gui/res/cardsfolder/upcoming/the_ninth_doctor.txt

```diff
@@ -0,0 +1,8 @@
+Name:The Ninth Doctor
+ManaCost:1 U R
+Types:Legendary Creature Time Lord Doctor
+PT:2/4
+K:Haste
+T:Mode$ Untaps | ConditionPhases$ Untap | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigAddPhase | TriggerDescription$ Into the TARDIS — Whenever CARDNAME becomes untapped during your untap step, you get an additional upkeep step after this step.
```

Comment:
the timing param you used does not exist 😢

Reply:
no, for which part do you think?

Reply:
`ActivationPhases$ ` and `UntapStep$ True ` don't seem to work as well. What's the param I'm missing ? 🤔

Reply:
Sorry to say it's just `Phase$`
For the second aspect I'd recommend just adding `+ActivePlayerCtrl` to the valid

Reply:
Thanks, will do